content/docs/command-reference/remote/modify.md

```diff
@@ -246,17 +246,19 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify myremote listobjects true
   ```
 
-- `sse` - server-side encryption algorithm to use (e.g. AES256, aws:kms). By
-  default, no encryption is used.
+- `sse` - server-side encryption algorithm to use: `AES256` or `aws:kms` (the
+  latter requires setting `sse_kms_key_id` as well). By default, no encryption
```

Comment:
Hm, I guess the docs have long said that `sse_kms_key_id` is required when using SSE-KMS, but I don't think that's true. AWS will automatically manage the keys for you if you don't provide one. See https://aws.amazon.com/premiumsupport/knowledge-center/s3-object-encryption-keys/.